src/content/1.7/release/release-a-new-version-of-prestashop.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Release a new version of PrestaShop
+weight: 4
+---
+
+# Release a new version of PrestaShop
+
+## Overview
+
+What does it mean, "to release a new version of "PrestaShop" ? Actually many different answers are eligible.
+
+You could say that "a new version of PrestaShop is released when there is a new release on the GitHub repository". Or you could say "a new version of PrestaShop is released when I can download a new version on prestashop.com". You could also say "a new version of PrestaShop is released when the auto-upgrade module allows to upgrade to this new version".
+
+So we should first put some words on all of these statements, to avoid any confusion.
+
+When we release a new version on PrestaShop:
+
+1. we create a new Build of PrestaShop, which is a ZIP archive which contains the software
+2. we make it available on GitHub as a GitHub release, and tag the branch used for the build to mark the exact code commit corresponding to this build
+3. we make it available on prestashop.com as a downloadable ZIP archive
+4. we activate multiple settings in PrestaShop Addons system so the PrestaShop API allows the new version of PrestaShop to download modules adapted to it
+5. we activate multiple settings in PrestaShop Internationalization API so it allows the new version of PrestaShop to download localization packs
+6. we update the XML stream being used by auto-upgrade module to monitor what versions of PrestaShop are available
+7. we create new Docker images for the new version
+
+## 1. Make a build
+
+If you want to create a new version of PrestaShop from its branch `1.7.6.x`, here is the process:
+
+1. Clone the repository, checkout the branch
+2. Update the code to acknowledge the new version:
+ - update version number in files `app/AppKernel.php` and `nstall-dev/install_version.php` (see the 1.7.6.6 [example](https://github.com/PrestaShop/PrestaShop/pull/19980))
+ - update the Changelog file `docs/CHANGELOG.txt` and the `CONTRIBUTORS.md` files with new content and new contributors (see the 1.7.6.6 [example](https://github.com/PrestaShop/PrestaShop/pull/20032))
+3. Use the embedded CLI [PrestaShop Release Creator](https://github.com/PrestaShop/PrestaShop/blob/develop/tools/build/README.md) to create the ZIP archive
+
+Example of Release Creator usage: `php tools/build/CreateRelease.php --version="1.7.6.6"`
+
+This allows you to create a new ZIP archive which contains the software. It can be installed and run, however the installation process might be unable to download new language packs and download any modules from Addons if the Addons and Translation systems do not acknowledge this version.
+
```

Comment:
```suggestion
If you want to remake the build of a release for comparison and repeatability purpose, ..
```
Not sure how to finish this sentence: "checkout the same tag that has been used", or something like that? what do you think?